### ISSUE TITLE
Add FAQ

### ISSUE THREAD

**Issue by therealklanni** [az-webdevs/azwebdevs.org#68]
Let's actually _add_ this after the CMS stuff is complete, but we should start discussing some FAQs to have, at least to start with.

Here are some I've thought of so far, but do chime in if you think of better verbiage.

Q. What is {az}webdevs?
A: [Basically the pitch from the "about" page, and if we want to go into more detail or something]

Q: Why Slack?
A: We certainly could have just started a channel on IRC, or used a different platform altogether, but the founder was familiar with and had some prior experience managing Slack teams. Slack provides some interesting integrations and other advantages over something like IRC that we wanted to take advantage of. Additionally, we wanted to have a private space where we can be free from harassment from spammers.

Q: Can I be an admin?
A: Yes and no. We want to try to keep the number of admins to a minimum, however if you're the admin of a local Meetup, the likelihood you will be made an admin in the Slack team increases drastically. Otherwise, if it makes sense to make you an admin for other reasons, feel free to speak to one of the current admins about it.

Q: Who owns the Slack team?
A: The owner is technically Kevin Lanni, who started the team originally. However, we want to promote a democratic approach to this community. This means that if the overwhelming majority of users are in favor of a certain change in how things are run, we will do whatever the community at large wants. This is your community, the owner and admins are just here to help facilitate and keep things running smoothly.

Q: Why can't I create my own channel/private group?
A: This decision was not made lightly, but we decided the best approach was to limit channel creation so that conversations don't become spread too thin (which was a problem we experienced with previous Slack teams). We will add more channels as needed and as the volume of conversations continues to grow. Members are always welcome to request a new channel, however requests will be approved only if it makes sense for the community as a whole.

Q: Why can't I post in #announcements?
A: We restricted posting to admins-only because this is the one channel that no member can opt out of. This restriction is in place to minimize unnecessary notifications, which we believe will help the majority of members feel more comfortable. With this in mind, we decided to use this channel only for the occasional important announcements.

@troym9731 please add your input

**Comment by therealklanni** [comment]
@troym9731 just in case you don't actually get notified by the mention in the description.

**Comment by troym9731** [comment]
Yup, I saw it. As of right now, no additional input. I think this is a good starter FAQ
